src/webgpu/shader/validation/uniformity/uniformity.spec.ts

```diff
@@ -777,6 +777,13 @@ const kPointerCases: Record<string, PointerCase> = {
     uniform: `never`,
     needs_deref_sugar: true,
   },
+  contents_rhs_pointer_swizzle: {
```

Comment:
Can you add a uniform value version of this case too?

Reply:
Done.